connector/src/main/scala/com/microsoft/kusto/spark/common/KustoOptions.scala

```diff
@@ -29,6 +29,10 @@ trait KustoOptions {
   // AAD application certificate password
   val KUSTO_AAD_APP_CERTIFICATE_PASSWORD: String = newOption("kutoAadAppCertPassword")
 
+  // Coordinates of a class that extends callable<String> with Serializeable and returns a token.
+  // The provider will be called for every request to the kusto service
+  val KUSTO_TOKEN_PROVIDER: String = newOption("tokenProvider")
```

Comment:
```suggestion
  val KUSTO_TOKEN_PROVIDER_CALLBACK: String = newOption("tokenProviderCallback")
```

This is a callback, right?

Reply:
Yea
Its actually class that extents callable

Reply:
Ok, then it's KUSTO_TOKEN_PROVIDER_CALLBACK_CLASSPATH